redfish/internal/controller/http/v1/handler/routes.go

```diff
@@ -80,14 +88,10 @@ const (
 	msgIDBaseGeneralError = "Base.1.22.0.GeneralError"
 )
 
-// metadataSearchPaths defines the search order for metadata.xml file locations
-var metadataSearchPaths = []string{
-	"redfish/internal/controller/http/v1/generated/metadata.xml",
-	"./redfish/internal/controller/http/v1/generated/metadata.xml",
-	"../../../redfish/internal/controller/http/v1/generated/metadata.xml",
-	"redfish/openapi/metadata.xml",
-	"./redfish/openapi/metadata.xml",
-}
+// metadataFilePath is the location of the metadata.xml file
+// The metadata.xml is generated by: make rf-metadata-generate
+// Source: redfish/openapi/infra/metadata-generate.py
+const metadataFilePath = "redfish/internal/controller/http/v1/generated/metadata.xml"
```

Comment:
This could be a problem when we deliver the binaries. The path could be in a different location and we need to distribute the metadata file as well as add options in the configuration file to point to the location.
The earlier approach of embed was good.